week3/Week3/src/RemoveDuplicateElements.java

```diff
@@ -0,0 +1,36 @@
+public class RemoveDuplicateElements {
+
+    public static void RemoveDuplicateElements(int arr1[], int size)
+    {
+        int arr2[] = new int [size];
+
+        for(int i = 0; i < size; i++)
```

Comment:
Please, test it again as it will throw an exception as  arr1[i+1] will be out in the last index beacause of this condition. also when you remove the dublicates it will leave its postion empty and you need to shift the array.